heareval/predictions/task_predictions.py

```diff
@@ -1208,6 +1213,7 @@ def task_predictions(
         seed_everything(42, workers=False)
 
     metadata = json.load(embedding_path.joinpath("task_metadata.json").open())
+    logger = logging.getLogger(metadata["task_name"])
```

Comment:
Instead of creating this again (which means it is created in twice places and thus the chance of bugs increase), why not just pass it in here?

I think my preference is that `logging.getLogger` is called only once per task, in runner.py, and everywhere else we pass it around the functions (with the parameter typed)